distributions/raspijamulus.sh

```diff
@@ -62,10 +65,10 @@ if [ -d "jack2" ]; then
   echo "The Jack2 directory is present, we assume it is compiled and ready to use. If not, delete the jack2 directory and call this script again."
 else
   git clone https://github.com/jackaudio/jack2.git
-  cd jack2
+  cd jack2 || exit
```

Comment:
@ann0see Needs a ` 1` as well, as @softins hinted on the other place. :)

Reply:
Yes. Just added it.